### ISSUE TITLE
Vulkan works in enhanced mode but not in basic mode

### ISSUE THREAD

**Issue by raven02** [jamesstringerparsec/Easy-GPU-PV#299]
OpenGL works prefectly on both enhanced and basic mode but not vulkan . Any insight?

**Comment by mitchellurgero** [comment]
You cannot do this with Hyper-V's console. Use Parsec like it explains in the readme friend :)

**Comment by mitchellurgero** [comment]
You'll get higher output (Hyper-V console limited to 30 FPS afaik) if you use parsec.

**Comment by bymodude** [comment]
I can only get Vulkan working (with NV 545.29 driver) when connected in Enhanced Session Mode, which is a shame because I'd like to use Sunshine with Moonlight to remote in but then Vulkan doesn't work.  Did anyone get Vulkan to work with Sunshine or Parsec?

**Comment by mitchellurgero** [comment]
Yeah Parsec will use the GPU passthrough but enhanced session uses RemoteFX which doesn't use the full passthrough of the card.

**Comment by bymodude** [comment]
Thanks for clearing that up, so in summary Vulkan is NOT working with gpu-partitioning, the issue title "Vulkan works" is wrong :)

**Comment by raven02** [comment]
Just tested with Windows Insider build 25947 with GPU passthrough , vulkan is fully working now with host driver using 537.13

**Comment by mitchellurgero** [comment]
Are you sure that is using the passed through GPU? From what I know, Hyper-V console & Enhanced session ONLY use the Hyper-V video adapter NOT the GPU (Hyper-V video adapter supports opengl and vulkan)

**Comment by raven02** [comment]
Probably it is using the hyper-v video adapter and that's why the performance is not quite good when compared with host.

**Comment by simonlsp** [comment]
> @mitchellurgero Probbaly using the hyper-v video adapter. I basically use the remoteplay in Steam client. The opengl and vulkan is about half of host . Wonder any way to use the true GPU passthrough

#342 
Workaround here, I solved this problem.
